Gemfile

```diff
@@ -2,9 +2,9 @@ source 'https://rubygems.org'
 ruby '2.5.0'
 
 ### Basic Framework
-gem 'rails', '5.2.2'
+gem 'rails', '~> 6.0.0.beta1'
 gem 'jbuilder' # DSL for building JSON view template
-gem 'haml-rails' # HTML template language, used instead of ERB
+gem 'haml-rails', git: "https://github.com/indirect/haml-rails.git" # HTML template language, used instead of ERB
```

Comment:
I assume this will be updated with a new release before or soon after Rails 6 final is released.

Reply:
Hopefully, this will be the case.